packages/shared/src/components/cards/PostMetadata.tsx

```diff
@@ -12,7 +25,9 @@ export default function PostMetadata({
   className?: string;
   children?: ReactNode;
 }): ReactElement {
+  const { flags } = useContext(FeaturesContext);
```

Comment:
Exposing this component to context and especially to the `FeaturesContext` will create tight coupling between them. I think it's better to accept this flag as property from the outside.